sceneformux/ux/src/main/java/com/google/ar/sceneform/ux/MovementListener.java

```diff
@@ -0,0 +1,13 @@
+package com.google.ar.sceneform.ux;
+
+/** Interface to visual when a {@link BaseTransformableNode} is moved. */
+public interface MovementListener {
```

Comment:
I relate this `Movement` with "transalation" and maybe "rotation" but not "scaling"; It sounds more like an interaction, or action, or even transformation listener to me. I guess is the same with the movement controller.